pkg/monitoring/vms/prometheus/prometheus.go

```diff
@@ -40,6 +40,18 @@ import (
 const statsMaxAge time.Duration = collectionTimeout + 2*time.Second // "a bit more" than timeout, heuristic again
 
 var (
+	// Formatter used to sanitize k8s metadata into metric labels
+	labelFormatter = strings.NewReplacer(".", "_", "/", "_", "-", "_")
+
+	// Preffixes used when transforming K8s metadata into metric labels
+	labelPreffix      = "kubernetes_vmi_label_"
+	annotationPreffix = "kubernetes_vmi_annotation_"
+
+	k8sLabels           = []string{}
```

Comment:
Doesn't this cause a data race?

Reply:
Could you take a look and point if this race condition was removed on this last commit?